src/styles/styles.js

```diff
@@ -1329,6 +1320,15 @@ const styles = {
         minHeight: variables.componentSizeNormal,
     },
 
+    chatFooter: {
```

Comment:
This is the same as `chatItemCompose`, just renamed. I just moved it so that it wasn't in between a number of other `chatItem*` styles.